lib/ui/views/installer/installer_view.dart

```diff
@@ -17,6 +17,8 @@ class InstallerView extends StatelessWidget {
       onViewModelReady: (model) => model.initialize(context),
       viewModelBuilder: () => InstallerViewModel(),
       builder: (context, model, child) => PopScope(
+        onPopInvoked: (bool didPop) => model.onWillPop(context),
+        canPop: false,
```

Comment:
The code seems skeptical to me.

Can you exit out of the screen when you finished patching?

Reply:
Yep, it does work. At least on my device.

Reply:
> Yep, it does work. At least on my device.

I'll revisit this later.:tm: